auth-web/devops/vaults.env

```diff
@@ -15,6 +15,7 @@ VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
 VUE_APP_AUTH_WEB_URL="op://web-url/$APP_ENV/auth-web/AUTH_WEB_URL"
 VUE_APP_DASHBOARD_URL="op://web-url/$APP_ENV/business/DASHBOARD_URL"
+VUE_APP_DOCUMENTS_UI_URL="op://web-url/$APP_ENV/documents-ui/DOCUMENTS_UI_URL"
```

Comment:
need these created in 1pass

Reply:
Copy that, wasn't sure if there was a global 1pass or per product etc. 
If there is one for auth-web, will definitely need it yessir. 

Also why i tagged Patrick Wei as well, a piece i don't have access to.